### PR TITLE
Python3 update

### PR DESCRIPTION
**Pull request by jhubig**
Minor update for python3 syntax.

### CODE REVIEW

**Comment by vchrisb** [comment]
I **think** `span: int = 0` is not the same as `def span(): int = 0`
The former is a type annotation https://www.python.org/dev/peps/pep-0526/ introduced in Python 3.6, while the later is defining a function.

File: e3dc/_e3dc.py [hunk 1]
```diff
@@ -600,10 +600,10 @@ def get_db_data(self, startDate: datetime.date = None, timespan: str = 'DAY', ke
                 }
         """
 
-        span: int = 0
+        def span(): int = 0
```
Comment (vchrisb) [inline e3dc/_e3dc.py:603]: I think span: int = 0 is not the same as def span(): int = 0
The former is a type annotation https://www.python.org/dev/peps/pep-0526/ introduced in Python 3.6, while the later is defining a function.

**Review by vchrisb — CHANGES_REQUESTED** [review]

**Comment by jhubig** [comment]
Okay. I got a syntax error at "span: int = 0". But I was using python 3.5.3. So means to have your library properly working I need to upgrade to python 3.6? Thanks and sorry for wasting your time with this pull request :|.

**Comment by vchrisb** [comment]
This function was introduced not too long ago and everyone who tested, was apparently using 3.6+
Removing the type annotation could be an option, as this is probably the only place where it is used.
If you have docker on your raspberry, you can also try out it in a python 3.6 container or you might want to check out https://github.com/vchrisb/e3dc-rest / https://hub.docker.com/repository/docker/vchrisb/e3dc-rest, depending on your usecase.

**Comment by vchrisb** [comment]
I'm closing this pull request but will incorporate the readme changes. 
Further discussion on Python 3.5 support should be handled in an issue.

**Comment by jhubig** [comment]
Okay. Thanks.

**Pull request closed without merging**